packages/editor/src/pages/Editor/components/Backstage/MySolutions/index.tsx

```diff
@@ -18,17 +21,29 @@ interface IProps {
 
 interface IState {
   filterQuery: string;
+  localStorageWarningIsVisible: boolean;
 }
 
 class MySolutions extends React.Component<IProps> {
-  state: IState = { filterQuery: '' };
+  state: IState = {
+    filterQuery: '',
+    localStorageWarningIsVisible: !localStorage.getItem(
```

Comment:
Should we have a more centralized system of configuration, rather than doing one-off keys in localStorage?

Reply:
we do have a more centralized location for the important keys. for this particular feature, only this page needs to know about it, so it didn't seem horrible to just define and use it here locally